network-observability.md

```diff
@@ -0,0 +1,84 @@
+# Network Observability
+
+This is a draft document, not implemented.
+
+## Current format
+
+[Network Observability](https://github.com/netobserv/) (or NetObserv) currently uses a custom format documented here: https://github.com/netobserv/network-observability-operator/blob/main/docs/flows-format.adoc.
+
+In the current NetObserv implementation, this format, or subsets of this format, can be retrieved as various data sources:
+- As logs, such as stored in Grafana Loki, using a JSON format
+- As metrics labels
+- As consumable JSON from Kafka
+
+## Relevant OTEL semantic conventions
+
+There is no perfect match between OTEL semantics and NetObserv format, however several conventions can be extrapolated to create a new model in OTEL style:
+
+- Traffic by source and destination: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/attributes.md#source-and-destination-attributes
+- Kubernetes resources: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/k8s.md
+
+These two specifications are combined by prefixing with "source." or "destination." any resource such as a Kubernetes Pod.
+
+A tradeoff is also done to avoid too much redundancy and its associated cost (increased message size, increased storage): the model being entirely related to networking, the `network.` prefix is omitted and considered as implicit. As a result, a field such as the network protocol is referred to as `protocol` instead of `network.protocol`.
+
+## Format proposal
+
+This new format is currently a proposal and is subject to change before being implemented in NetObserv.
```

Comment:
Is there a link to the proposal or to the primiray source of truth?

Reply:
This is the original draft as we haven't formally planned transitioning to this model yet.